go/tasks/pluginmachinery/flytek8s/pod_helper.go

```diff
@@ -202,6 +202,9 @@ func ConvertPodFailureToError(status v1.PodStatus) (code, message string) {
 					c.LastTerminationState.Terminated.ExitCode,
```

Comment:
Should we change the loop to be:
`for _, c := range append(status.InitContainerStatuses, status.ContainerStatuses...) {`

and just keep it as one loop?

Reply:
the messaging is slightly different based on the type of container so i can't think of an elegant way to pass that along